examples/error-type-intersection/error_type_intersection.md

```diff
@@ -1,7 +1,9 @@
 # Type intersection for error types
 
-You can define an error type that is both a subtype of a `distinct` error type and has additional constraints on the detail fields using intersection types.
+If you intersect two `error` types, the resulting type's detail type is the intersection of the detail types of both types. Furthermore, if any of the types being intersected is a distinct type, then the resultant type's type ID set includes all the type IDs of that type. Thus it is a subtype of both types and this is how you create subtypes of `distinct` error types.
```

Comment:
```suggestion
If you intersect two `error` types, the resulting type's detail type is the intersection of the detail types of both types. Furthermore, if any of the types being intersected is a distinct type, then the resultant type's type ID set includes all the type IDs of that type. Thus it is a subtype of both types. This way, you can create an error type that is a subtype of multiple distinct types and also use a more specific detail type.
```

Reply:
Addressed in c19adcc407f68b124fae5f39fe8a901d22566377